daemon/cluster/cluster.go

```diff
@@ -509,6 +509,14 @@ func (c *Cluster) stopNode() error {
 	return nil
 }
 
+func atLeastOneMoreNodeReachable(reachable, unreachable int) bool {
```

Comment:
I don't think this name accurately describes what it's doing. I believe it's checking for a situation where removing the manager would cause a loss of quorum.